packages/shared/lib/tests/__mocks__/account.mock.ts

```diff
@@ -129,11 +130,11 @@ export class AccountMock implements IAccount {
         return Promise.resolve([])
     }
 
-    listOutputs(): Promise<[]> {
+    listOutputs(filterOptions?: FilterOptions): Promise<[]> {
         return Promise.resolve([])
     }
 
-    listUnspentOutputs(): Promise<[]> {
+    listUnspentOutputs(filterOptions?: FilterOptions): Promise<[]> {
```

Comment:
we could use this function in `prepareShimmerClaimingAccount.ts` Line ` 30`

Reply:
Not as far as I can see. `FilterOptions` is only useful to specify a timestamp range